artman/cli/main.py

```diff
@@ -37,6 +37,7 @@
 from artman.cli.support import parse_github_credentials
 from artman.cli.support import parse_local_paths
 from artman.cli.support import resolve
+from artman.cli.support import select_git_repo
```

Comment:
I have updated the imports from `artman.cli.support`. I haven't changed the importing of the `artman.utils.logger` module, as I think any changes there should also update the module itself given that it is designed to be imported in this way.